src/wp-includes/script-loader.php

```diff
@@ -2987,7 +2969,7 @@ function wp_get_inline_script_tag( $data, $attributes = array() ) {
 	 * @see https://www.w3.org/TR/xhtml1/#h-4.8
 	 */
 	if (
-		! $is_html5 &&
+		! current_theme_supports( 'html5', 'script' ) &&
```

Comment:
The `$is_html5` variable checked for theme support **plus** `is_admin()`. With r61411, if I activate a theme such as Twenty Eleven, I now get `CDATA` for scripts in the admin.
```
<script id="utils-js-extra">
/* <![CDATA[ */
var userSettings = {"url":"/svn/src/","uid":"1","time":"1766767047","secure":""};
//# sourceURL=utils-js-extra
/* ]]> */
</script>
```

Of course, this should not need fixing if #10660 removes `CDATA` from _all_ scripts.

Reply:
Thanks for spotting that!

I'm happy addressing it with a small fix: https://github.com/WordPress/wordpress-develop/pull/10666.

But I do expect https://github.com/WordPress/wordpress-develop/pull/10660 to remove all the CDATA wrappers soon.

Reply:
Handled in r61414.